src/main/java/bio/terra/workspace/service/datareference/model/DataReferenceType.java

```diff
@@ -18,8 +22,12 @@
 
   static {
     typeMap.put(DataReferenceType.DATA_REPO_SNAPSHOT, ReferenceTypeEnum.DATA_REPO_SNAPSHOT);
+    typeMap.put(DataReferenceType.GOOGLE_BUCKET, ReferenceTypeEnum.GOOGLE_BUCKET);
```

Comment:
And this map is overkill. Maybe I will write you an "enum" class that will use a list search - which is generally faster than a hash for lists smaller than 10 items...

I know, it probably isn't important and it is all static. If we were running on a PDP-11/70 you would thank me :)

Reply:
https://skn.noip.me/pdp11/pdp11.html